V2/time_tracker/_infrastructure/_db.py

```diff
@@ -2,6 +2,8 @@
 
 from . import _config
 
+_TEST_DIALECT = "sqlite"
```

Comment:
I don't think we should hardcode the sqlite dialect. What are we going to do when we want to connect to postgres?

Reply:
si, esto es para las pruebas, al parecer sqlite no maneja las claves foraneas sin este comando `self.connection.execute("pragma foreign_keys=ON")`